tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
update to es2022, pls

Reply:
because compile code need to use module that is the last version stable in 2022 Javascript. Module es5(2016) is too old !